### PR TITLE
Localize FXIOS-4800 [v106] Remove more unused strings

### PR DESCRIPTION
**Pull request by flodolo**
Mostly focusing on strings with last update unknown 
https://gist.github.com/flodolo/b859884d7f700de678839e1c50a96773#file-unused-strings-in-fx-ios

https://github.com/mozilla-mobile/firefox-ios/issues/11665 [FXIOS-4800](https://mozilla-hub.atlassian.net/browse/FXIOS-4800)

### CODE REVIEW

**Comment by flodolo** [comment]
A bit larger than I hoped, but after this we're left with about 20 strings added between 96 and 103.

File: Client/Frontend/Strings.swift [hunk 1]
```diff
@@ -69,24 +63,6 @@ extension String {
...
-                "Bookmarks.Actions.BookmarkAllTabs",
-                value: "Bookmark All Tabs",
-                comment: "A label indicating the action of bookmarking all currently open non private tabs.",
-                lastUpdated: .v96)
-            public static let BookmarkCurrentTab = MZLocalizedString(
-                "Bookmarks.Actions.BookmarkCurrentTab",
-                value: "Bookmark Current Tab",
-                comment: "A label indicating the action of bookmarking the current tab.",
-                lastUpdated: .v96)
-        }
```
Comment (lmarceau) [inline Client/Frontend/Strings.swift:88]: @dnarcese do we plan on working on those bookmarks changes? Can those be removed?

**Review by lmarceau — COMMENTED** [review]
Just one question for Daniela, otherwise LGTM

File: Client/Frontend/Strings.swift (hunk 1 above)
Comment (dnarcese) [reply Client/Frontend/Strings.swift:88]: Lets remove them

**Review by lmarceau — APPROVED** [review]

**Pull request merged**